platforms/corda-ent/configuration/roles/helm_component/templates/idman.tpl

```diff
@@ -15,49 +15,54 @@ spec:
     nodeName: {{ node_name }}
     metadata:
       namespace: {{ component_ns }}
-    storage:
-      name: {{ storageclass }}
-    replicas: 1
     image:
       initContainerName: {{ init_container_name }}
-      imagepullsecret: {{ image_pull_secret }}
-    dockerImage:
-      name: {{ idman_image_name }}
-      tag: {{ idman_image_tag }}
+      idmanContainerName: {{ idman_image_name }}:{{ idman_image_tag }}
       pullPolicy: Always
       imagePullSecret: {{ image_pull_secret }}
+    storage:
+      name: {{ storageclass }}
+      memory: 64Mi
     acceptLicense: YES
     vault:
       address: {{ vault_addr }}
-      certsecretprefix: {{ vault_cert_secret_prefix }}
+      certSecretPrefix: {{ vault_cert_secret_prefix }}
       role: {{ vault_role }}
-      authpath: {{ auth_path }}
-      serviceaccountname: {{ vault_serviceaccountname }}
-    ambassador:
-      external_url_suffix: {{ external_url_suffix }}
-    volume:
-      baseDir: /opt/corda
+      authPath: {{ auth_path }}
+      serviceAccountName: {{ vault_serviceaccountname }}
     service:
-      port: {{ idman_port }}
-    serviceInternal:
-      port: 5052
-    serviceRevocation:
-      port: 5053
-    shell:
-      sshdPort: 2222
-      user: {{ ssh_username }}
-      password: {{ ssh_password }}
+      external:
+        port: {{ idman_port }}
+      internal:
+        port: 5052
+      revocation:
+        port: 5053
+      shell:
+        sshdPort: 2222
+        user: {{ ssh_username }}
+        password: {{ ssh_password }}
     database:
       driverClassName: "org.h2.Driver"
       url: "jdbc:h2:file:./h2/identity-manager-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
       user: {{ db_username }}
       password: {{ db_password }}
       runMigration: "true"
-    cordaJarMx: 1
-    healthCheckNodePort: 0
-    jarPath: bin
-    configPath: etc
-    sleepTimeAfterError: 120
-    healthcheck:
-      readinesscheckinterval: 10
-      readinessthreshold: 15
+    config:
+      volume:
+        baseDir: /opt/corda
+      jarPath: bin
+      configPath: etc
+      cordaJar:
+        memorySize: 512
+        unit: M
+        resources:
+          limits: 512M
+          requests: 512M
+      replicas: 1
+      sleepTimeAfterError: 120
+    healthCheck:
+      readinessCheckInterval: 10
+      readinessThreshold: 15
+      nodePort: 0
```

Comment:
remove nodePort: 0 as this is confusing, rather not add the nodePort at all and it will use the empty one from values.yaml